samples/ZoomSDKSampleApp/ZoomSDKSampleApp.Android/ZoomSDKSampleApp.Android.csproj

```diff
@@ -90,6 +91,14 @@
       <Project>{AA0EE7FA-AE09-440D-A0B2-F254D6EEA592}</Project>
       <Name>ZoomSDKSampleApp</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\src\Droid\ZoomCommonLib\ZoomCommonLib.csproj">
+      <Project>{8F896C4F-56FE-4237-9373-D47F564D7065}</Project>
+      <Name>ZoomCommonLib</Name>
+    </ProjectReference>
```

Comment:
@icalderond Does the sample app need to reference the ZoomCommonLib directly itself? MobileRTC_Droid already references ZoomCommonLib itself so it should be fine without? Or am I mistaken?

Reply:
It seems is not possible keep only RTC .aar file because when ia removed the another one (ZoomCommonLib.aar) the project show this error.

![image](https://user-images.githubusercontent.com/5691112/197752423-e5971840-ea85-4a8a-9b55-14785d32822e.png)

Reply:
Ok cool